docs/install.md

```diff
@@ -199,6 +199,21 @@ By default the Dashboard accesses resources and performs actions in the cluster
 
 Typically when configuring impersonation you would have the proxy forward its ServiceAccount token in the `Authorization` header, and details of the user and groups in the `Impersonate-User` and `Impersonate-Group` headers respectively. See the docs of your chosen solution for details.
 
+When using a reverse proxy, with impersonation headers or the user's account, you should remove the dashboard's privileges to better maintain a "least privileged" approach.  This will make it less likely that the dashboard's `ServiceAccount` will be abused:
```

Comment:
nit:

```suggestion
When using a reverse proxy, with impersonation headers or the user's account, you should remove the Dashboard's privileges to better maintain a "least privileged" approach.  This will make it less likely that the Dashboard's `ServiceAccount` will be abused:
```